### DISCUSSION TITLE
LFRic Apps vn1.2 Release

### DISCUSSION THREAD

**Discussion by ericaneininger** [MetOffice/simulation-systems#274] [Announcements]
Dear all,

I'm happy to announce that LFRic Apps vn1.2 now been released and is available for use. Many thanks to all developers and reviewers for their efforts and contributions.

Erica on behalf of SSD team.

**Comment by jennyhickson** [comment]
The Code Review deadline for vn2.1 will be 24th January. Please plan your changes and developments accordingly, thanks 👍
